assets/sass/components/global/_googlesitekit-gathering-data-notice.scss

```diff
@@ -23,7 +23,7 @@
 		text-align: center;
 
 		p {
-			color: $c-jumbo;
+			color: $c-silver-2;
```

Comment:
@jimmymadon That change will cause accessibility to fail. True it's as per the designs in Figma but it does not pass accessibility test as seen here: https://webaim.org/resources/contrastchecker/?fcolor=CCCCCC&bcolor=FFFFFF

I suggest we revert that change.

Reply:
Ahh right - nice catch. Reverting. We should add a note to the Figma mock 😄 or someone else will do this again!